Sources_v3/WebSocketClient/ChatClientConnectionStatus.swift

```diff
@@ -4,12 +4,49 @@
 
 import Foundation
 
-public typealias ConnectionId = String
+// `ChatClientConnectionStatus` is just a simplified and friendlier wrapper around `WebSocketConnectionState`.
+
+/// Describes the possible states of the client connection to the servers.
+public enum ChatClientConnectionStatus: Equatable {
```

Comment:
Maybe better to name it as `ClientConnectionStatus`, because `ChatClient` is just a type alias for `Client`?

Reply:
I renamed it to `ConnectionStatus` in `e75bc67`. It's clear enough since it's the only public type like that we expose.